Xamarin.PropertyEditing.Windows/PropertyPresenter.cs

```diff
@@ -112,6 +113,11 @@ public override void OnApplyTemplate ()
 
 			this.variationRow = (RowDefinition) GetTemplateChild ("variationRow");
 			this.variationsList = GetTemplateChild ("variationsList") as ItemsControl;
+
+			this.propertyButton = (PropertyButton) GetTemplateChild ("propertyButton");
+
+			this.propertyContainer = (Border) GetTemplateChild ("propertyContainer");
+			this.propertyContainer.AddHandler (Border.PreviewKeyDownEvent, new KeyEventHandler (PropertyContainer_PreviewKeyDown));
```

Comment:
there is any mechanism to unsuscribe it?

Reply:
Since the template never changes, OnApplyTemplate is only called once and there shouldn't be a need to call RemoveHandler. But I added a comment, to make that clearer. BrushEditorControl has the same pattern.